config/_default/menus/menus.en.yaml

```diff
@@ -7325,14 +7325,14 @@ api:
         - ListPermissions
       unstable: []
       order: 1
-  - name: Security Monitoring
-    url: /api/latest/security-monitoring/
-    identifier: security-monitoring
+  - name: Cloud SIEM
+    url: /api/latest/cloud-siem/
+    identifier: cloud-siem
     generated: true
```

Comment:
Any of these nav items that say `generated: true` get built/rebuilt from the api spec files so changes here can be lost. This might be why you had some broken links after merging in master.